### ISSUE TITLE
Reduce response from the readRequests API.

### ISSUE THREAD

**Issue by reutenkoivan** [CarSnoop-Inc/carsnoop-issues#50]
### Expected response type:

```typescript
type ReadRequestsResponse = Array<{
  id: string
  status: string
  year: string
  make: string
  model: string
  trim: string
  exterior_color: string
  interior_color: string
  expires_on: string
  trade_in_vehicle: null | {
    id: string
    document_content: null | string
    year: number
    make: string
    model: string
    trim: string
  }
  transactions: Array<{
    id: string
    status: string
    vehicle: {
      id: string
      document_content: null | string
      year: number
      make: string
      model: string
      trim: string
      mileage: number
      retail_value: number
      shipping_cost: number
      distance: string
    }
  }>
}>
```

**Comment by ramziddinAndersen** [comment]
Yes, It is ok for mobile

**Comment by reutenkoivan** [comment]
Resolves #49
